keras/saving/object_registration.py

```diff
@@ -38,7 +38,7 @@ class CustomObjectScope:
     ```
 
     Args:
-        custom_objects: Dictionary of `{name: object}` pairs.
+        custom_objects: Dictionary of `{str: object}` pairs. Where the str key is the name.
```

Comment:
```
, where the `str` key is the object name.
```

Reply:
That turned the output into `Mapping[str, object, pairs, where, the]`. I changed the text though to match yours; just with a full-stop. Also fixed a minor typo in a function below it.